pallets/frequency-tx-payment/src/tests/mock.rs

```diff
@@ -52,7 +54,7 @@ impl Get<frame_system::limits::BlockWeights> for BlockWeights {
 				weights.base_extrinsic = ExtrinsicBaseWeight::get().into();
 			})
 			.for_class(DispatchClass::non_mandatory(), |weights| {
-				weights.max_total = Weight::from_ref_time(1024).set_proof_size(u64::MAX).into();
+				weights.max_total = Some(NORMAL_DISPATCH_RATIO * MAXIMUM_BLOCK_WEIGHT);
```

Comment:
Remind why we needed to change this one?

Reply:
The original `max_total` weight of 1024 does not work on some tests because the new fee gets computed with the static weight (90_764_000, `mock.rs` previously used 5), so the `max_total` needs to be larger or else part of the fee computation comes back as 1024 incorrectly. I used the actual runtime code from `lib.rs` to get the `max_total` weight.